src/components/theme/View/View.jsx

```diff
@@ -220,8 +222,65 @@ class View extends Component {
           {this.props.content.language && (
             <html lang={this.props.content.language.token} />
           )}
-          <title>{this.props.content.title}</title>
-          <meta name="description" content={this.props.content.description} />
+          <title>
+            {this.props.content.seo_title || this.props.content.title}
+          </title>
+          <meta
+            name="description"
+            content={
+              this.props.content.seo_description ||
+              this.props.content.description
+            }
+          />
+
+          <meta
+            property="og:title"
+            content={
+              this.props.content.opengraph_title ||
+              this.props.content.seo_title ||
+              this.props.content.title
+            }
+          />
+          <meta
+            property="og:url"
+            content={
+              this.props.content.seo_canonical_url ||
+              toPublicURL(this.props.content['@id'])
+            }
+          />
+          {this.props.content.image?.scales?.large?.download ||
+            (this.props.content.opengraph_image?.scales?.large?.download && (
+              <>
```

Comment:
this throws errors.. each <meta> it must not be enclosed in a fragment

Reply:
👍